subsystems/win32/csrsrv/server.c

```diff
@@ -153,12 +153,11 @@ CsrLoadServerDll(IN PCHAR DllString,
             RtlInitUnicodeString(&ErrorString, L"Default Load Path");
 
             /* Send a hard error */
-            NtRaiseHardError(Status,
-                             2,
-                             3,
-                             Parameters,
-                             OptionOk,
-                             &Response);
+            Status2 = NtRaiseHardError(Status, 2, 3, Parameters, OptionOk, &Response);
+            if (!NT_SUCCESS(Status2))
```

Comment:
It doesn't really make sense to look for the returned status of NtRaiseHardError, if it's just for dprinting stuff in the debug log. The idea here was to just raise the hard-error, and then the code will continue execution.

Reply:
Reverted, then.